tests/ai_safety/guardrails/utils.py

```
@@ -385,3 +385,53 @@ def send_and_verify_standalone_detection(
+@retry(wait_timeout=60, sleep=5)
+def check_guardrails_traces_in_tempo(tempo_traces_service_portforward: str):
+    """
+    Check for guardrails traces in Tempo.
+
+    Args:
+        tempo_traces_service_portforward: The Tempo traces service port-forward URL
+
+    Returns:
+        The traces data if found, False otherwise
+    """
+    services = requests.get(f"{tempo_traces_service_portforward}/api/services").json().get("data", [])
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)